sktime/forecasting/arch.py

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+
+"""Implements Generalized Autoregressive Conditional Heteroskedasticity(GARCH)."""
+
+__author__ = ["Vasudeva-bit"]
+
+from sktime.forecasting.base import BaseForecaster
+from sktime.utils.validation._dependencies import _check_soft_dependencies
+
+_check_soft_dependencies("arch", severity="warning")
+
+
+class ARCH(BaseForecaster):
+    """Directly interfaces ARCH models from python package arch.
+
+    Generalized Autoregressive Conditional Heteroskedasticity (GARCH)
+    model for forecasting votility in high frequency time series data.
+
+    ARCH models are a popular class of volatility models that use observed values
+    of returns as volatility shocks.
+
+    Implements Generalized Autoregressive Conditional Heteroskedasticity
+    (GARCH) models introduced and implemented in [1]_, [2]_, and [3]_ to
+    forecast the volatility in high frequency time series data, Implemented
+    using python package for ARCH models called "arch".
+
+    A complete ARCH model is divided into three components:
+        a mean model, e.g., a constant mean or an ARX;
+        a volatility process, e.g., a GARCH or an EGARCH process; and
+        a distribution for the standardized residuals.
+
+    Parameters
+    ----------
+    y : {np.ndarray, Series, None}
+        The dependent variable
+    x : {np.ndarray, DataFrame}, optional
+        Exogenous regressors.  Ignored if model does not permit exogenous
+        regressors.
+
+    Hyper-parameters
```

Comment:
there should only be a sectiion ´Parameters´

Reply:
Hmm okay, I will put everything under `parameters`.